cmd/cli.go

```diff
@@ -114,7 +114,12 @@ func runQuery(queryCmd string) {
 func getQueryConfigs(dbConn *qemodels.DBConnection) *qemodels.QueryConfig {
 	createLog := func(query string) {
 		queryLog := models.NewQueryLog(dbConn.ID, query)
-		go dao.DBQueryLog.CreateDBQueryLog(queryLog)
+		go func() {
```

Comment:
The code added is same as code removed. Except, the first code was just one line.
Doesn't make sense to write this.

Reply:
Yes, your right.